install_tcsh.sh

```diff
@@ -62,8 +62,8 @@ setenv VENV $ALIGN_HOME/general
 cd $ALIGN_HOME
 python3 -m venv $VENV
 source $VENV/bin/activate
```

Comment:
This might be the problem. If you are using CSH, then we need to activate the enviroment using the CSH specific script.
This line should be `source $VENV/bin/activate.csh`

@kkunal1408 Please make this change.